### ISSUE TITLE
Importing fails on Ubuntu 18.04

### ISSUE THREAD

**Issue by MartinPernus** [IntelVCL/Open3D#498]
Hi,

I realize Open3D is not tested on 18.04 but I'm trying to use it nevertheless. When doing 
`import open3d`
I get an error saying: 
`ImportError: libGLEW.so.1.13: cannot open shared object file: No such file or directory`
 
When searching with `sudo find / -name 'libGLEW*'`, following shared objects are found:
`/usr/lib/x86_64-linux-gnu/libGLEWmx.so.1.13`
`/usr/lib/x86_64-linux-gnu/libGLEW.so.2.0.0`
`/usr/lib/x86_64-linux-gnu/libGLEWmx.so.1.13.0`
`/usr/lib/x86_64-linux-gnu/libGLEW.so`
`/usr/lib/x86_64-linux-gnu/libGLEW.so.2.0`

I tried making a symbolic link linking to some of these, however this results in segmentation fault when importing. Is there any solution for this?

**Comment by takanokage** [comment]
Hi,

We've added a Ubuntu 18.04 based [docker solution](https://github.com/IntelVCL/Open3D/tree/master/util/docker/open3d-xvfb) a while back. Maybe that recipe can help you figure out your native installation.

**Comment by yxlao** [comment]
Previously the pip package that we released wasn't compiling the dependencies from source as static libs, thus it relies on system shared libraries. This was fixed in our v0.3.0 release.

Our current PyPI and Conda release is built with the following flags
```
cmake .. -DWITH_OPENMP=ON -DBUILD_EIGEN3=ON -DBUILD_GLEW=ON \
         -DBUILD_GLFW=ON -DBUILD_JPEG=ON -DBUILD_JSONCPP=ON \
         -DBUILD_PNG=ON -DBUILD_PYBIND11=ON -DBUILD_PYTHON_MODULE=ON \
         -DBUILD_TINYFILEDIALOGS=ON
```

It shall work out-of-the-box for Ubuntu 18.04, shown as follows:
```
➜  ~ virtualenv /home/yixing/venvs/fresh3.6 -p python3.6
Running virtualenv with interpreter /usr/bin/python3.6
Using base prefix '/usr'
New python executable in /home/yixing/venvs/fresh3.6/bin/python3.6
Also creating executable in /home/yixing/venvs/fresh3.6/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.

➜  ~ source /home/yixing/venvs/fresh3.6/bin/activate
(fresh3.6) ➜  ~ pip install open3d-python
Collecting open3d-python
  Downloading https://files.pythonhosted.org/packages/df/e8/6497d43442f5ff0da42c1f3f2db075e10a87cadedd0cd23150443708def1/open3d_python-0.3.0.0-py2.py3-none-any.whl (17.2MB)
    100% |████████████████████████████████| 17.2MB 890kB/s 
Collecting numpy (from open3d-python)
  Downloading https://files.pythonhosted.org/packages/fe/94/7049fed8373c52839c8cde619acaf2c9b83082b935e5aa8c0fa27a4a8bcc/numpy-1.15.1-cp36-cp36m-manylinux1_x86_64.whl (13.9MB)
    100% |████████████████████████████████| 13.9MB 1.2MB/s 
Installing collected packages: numpy, open3d-python
Successfully installed numpy-1.15.1 open3d-python-0.3.0.0

(fresh3.6) ➜  ~ which python             
/home/yixing/venvs/fresh3.6/bin/python

(fresh3.6) ➜  ~ python -c "from open3d import *"

(fresh3.6) ➜  ~ python -c "import open3d; print(open3d)"
<module 'open3d' from '/home/yixing/venvs/fresh3.6/lib/python3.6/site-packages/open3d/__init__.py'>

(fresh3.6) ➜  ~ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
```
